src/coresyftools/coresyftools/tool.py

```diff
@@ -3,15 +3,36 @@
 
 import shutil
 
+import string
+
 import zipfile
 
 import logging
 
 from manifest import get_manifest
 from argument_parser import CoReSyFArgumentParser
 
+from sarge import run, shell_format, Capture
+
 TMP_DIR = os.path.abspath('tmp')
 
+class MissingCommandPlaceholderForOption(Exception):
+
+    def __init__(self, option_identifier):
+        super(MissingCommandPlaceholderForOption, self).__init__(
+            'Defined {} identifier, but not used it in the command template.'
```

Comment:
`Identifier {} was defined, but not used in the command template.`

Reply:
Replaced.